src/hooks/useSwapDerivedOutputs.ts

```diff
@@ -13,7 +13,7 @@ import { useQuery } from '@tanstack/react-query';
 import { useCallback, useMemo, useState } from 'react';
 // DO NOT REMOVE THESE COMMENTED ENV VARS
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-import { IS_TESTING } from 'react-native-dotenv';
+import { IS_APK_BUILD, IS_TESTING } from 'react-native-dotenv';
```

Comment:
DO NOT REMOVE THESE COMMENTED ENV VARS
(my prettier removed it my bad 😬)

have to make my config work gonna spend the time now